docs/analysis.rst

```diff
@@ -10,25 +10,143 @@ the final free energy output that you get. In theory, we could just print out
 "Free energy of Binding =  XX.XXX +- Y.YYY kcal/mol," but that alone does not tell you anything about the quality of
 your free energy estimate. We try to provide guides and built-in analysis to help understand the numbers YANK provides.
 
-The analysis framework comes with a full feature API, command line auto-execution, and
-`Jupyter Notebooks <http://jupyter.org/>`_ to help the user analyze their data programmatically, automatically, or
-visually, depending on their choice.
+The analysis framework comes three ways to help the user analyze their data (use these links to **jump to usage**):
 
-**************
-Analyzing data
-**************
+* :ref:`Command line auto-execution <auto_analyze>`
+* :ref:`Cisually with Jupyter Notebooks<visual_analyze>`
+* :ref:`Full feature API <programmatic_analyze>`
 
-To perform a simple analysis of the data generated by YANK, you can use the ``yank analyze`` facility, even when YANK is running.
 
-.. code-block:: bash
+Analysis Theory
+---------------
+
+We run YANK simulations through  improve the estimate in free energy and other thermodynamic properties. We document them in
```

Comment:
Something is missing here: "We run YANK simulations through  improve the estimate"